web-ng/src/app/components/card-view-project/card-view-project.component.html

```diff
@@ -10,7 +10,7 @@ <h1 fxFlex style="padding: 16px;"> Recent Projects</h1>
                   <mat-card-header>
                     <mat-card-title>
                     <button mat-icon-button color="primary">
-                      <mat-icon>home</mat-icon>
+                      <mat-icon [ngStyle]="{'color':'white'}">add</mat-icon>
```

Comment:
I believe the color should be inherited from the color on the `mat-icon-button`. Perhaps changing the color of the button will fix the icon color?

Reply:
updated